src/NServiceBus.Core/Pipeline/BehaviorContext.cs

```diff
@@ -7,7 +7,7 @@ namespace NServiceBus
     /// <summary>
     /// Provides base context for behavior context implementations.
     /// </summary>
-    public abstract class BehaviorContext : ContextBag, IBehaviorContext
+    abstract class BehaviorContext : ContextBag, IBehaviorContext
```

Comment:
Should we leave this public in cases where they want to implement our interfaces and use our base class for convenience?